### PR TITLE
v1.0

### PR DESCRIPTION
**Pull request by bcaldwell**

### CODE REVIEW

File: git-repo-mirror.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  runCmd ("git", []string{"fetch", "-p","origin"}, hook.Name)
+  runCmd ("git", []string{"push", "--mirror"}, hook.Name)
+}
+
+
+func parseYamlConfig () []webhook{
+  hooks := make([]webhook, 0)
+
+  data, err := ioutil.ReadFile("/etc/git-repo-mirror/config.yml")
+  // data, err := ioutil.ReadFile("config.yml")
```
Comment (benjamincaldwell) [inline git-repo-mirror.go]: Make env config

**Comment by karanthukral** [comment]
Surprisingly a short file. Pretty cool

**Comment by karanthukral** [comment]
Nice refactor

**Comment by bcaldwell** [comment]
Thanks. It seems to be working now which is awesome.

File: Godeps/Readme [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This directory tree is generated automatically by godep.
+
+Please do not edit.
+
+See https://github.com/tools/godep for more information.
```
Comment (benjamincaldwell) [inline Godeps/Readme]: Finish documentation

File: README.md [hunk 3]
```diff
@@ -1 +1,5 @@
-# git-repo-mirror
\ No newline at end of file
+# git-repo-mirror
+
+## Setting up on a server
+### SSH access
+For this to work the machine or container that it is running in must have push and read access for the repos. For read access great deploy keys (works for both github and gitlab). For read access either create a new deploy user or use deploy keys with push access (works in github)
```
Comment (bcaldwell) [inline README.md:34]: finish documentation

**Comment by kevintpeng** [comment]
This is really cool actually, nice use-case for go. My one week of go experience doesn't see anything wrong with this 🎉

**Comment by bcaldwell** [comment]
🚢

**Pull request merged**